src/main/kotlin/org/lfenergy/gxf/protocol/adapter/oslp/mikronika/device/communication/sockets/ServerSocketMessageProcessor.kt

```diff
@@ -46,11 +46,17 @@ class ServerSocketMessageProcessor(
     private fun getStrategyFor(message: Message): ReceiveStrategy {
         with(message) {
             return when {
-                hasRegisterDeviceRequest() -> strategyFactory.getStrategy("RegisterDeviceStrategy")
-                hasConfirmRegisterDeviceRequest() -> strategyFactory.getStrategy("ConfirmRegisterDeviceStrategy")
-                hasEventNotificationRequest() -> strategyFactory.getStrategy("EventNotificationRequestStrategy")
+                hasRegisterDeviceRequest() -> strategyFactory.getStrategy(REGISTER_DEVICE_STRATEGY)
+                hasConfirmRegisterDeviceRequest() -> strategyFactory.getStrategy(CONFIRM_REGISTER_DEVICE_STRATEGY)
+                hasEventNotificationRequest() -> strategyFactory.getStrategy(EVENT_NOTIFICATION_STRATEGY)
                 else -> error("Unexpected request message: $message")
             }
         }
     }
+
+    companion object {
+        const val REGISTER_DEVICE_STRATEGY = "RegisterDeviceStrategy"
+        const val CONFIRM_REGISTER_DEVICE_STRATEGY = "ConfirmRegisterDeviceStrategy"
+        const val EVENT_NOTIFICATION_STRATEGY = "EventNotificationRequestStrategy"
```

Comment:
I think I would prefer to have these constants in a different location, probably within the strategy package? (in order to prevent the `ServerSocketMessageProcessor.Companion` imports in the strategy classes)